Configuration.EntityFrameworkCore/src/SeoYarp.Configuration.EntityFrameworkCore/Extensions/ModelBuilderExtensions.cs

```diff
@@ -52,13 +53,101 @@ public static ModelBuilder ConfigureProxyRoutes(this ModelBuilder modelBuilder,
                     transform.Property(x => x.Data).HasJsonConversion();
                 });
             });
+
+
+            modelBuilder.Entity<Cluster>(cluster =>
+            {
+                cluster.ToTable(storeOptions.Clusters);
+
+                cluster.HasKey(c => c.Id);
+
+                cluster.Property(c => c.Id)
```

Comment:
I think it's better to have an auto increment int Id as primary key. We could store the actual cluster Id in ClusterId column.